src/setupTests.ts

```diff
@@ -40,35 +39,36 @@ jest.mock("@demo/core/constants/appConfig", () => ({
 		CF_TEMPLATE: "https://www.example.com?region=us-east-1",
 		APPLE_APP_STORE_LINK: "",
 		GOOGLE_PLAY_STORE_LINK: "",
-		COUNTRY_EVALUATION_URL: ""
+		COUNTRY_EVALUATION_URL: "",
+		APP_VERSION: "",
+		NL_BASE_URL: "",
+		NL_API_KEY: ""
 	},
 	PERSIST_STORAGE_KEYS: {
-		LOCAL_STORAGE_PREFIX: "amazon-location-web-demo_",
-		AMPLIFY_AUTH_DATA: "AmplifyAuthData",
-		AMPLIFY_MAP_DATA: "AmplifyMapData",
-		PERSISTED_DATA: "PersistedData",
-		GEO_LOCATION_ALLOWED: "GeoLocationAllowed",
-		SHOULD_CLEAR_CREDENTIALS: "ShouldClearCredentials",
+		LOCAL_STORAGE_PREFIX: "amazon-location_",
+		AMPLIFY_AUTH_DATA: "amplifyAuthData",
+		AMPLIFY_MAP_DATA: "amplifyMapData",
+		PERSISTED_DATA: "persistedData",
+		GEO_LOCATION_ALLOWED: "geoLocationAllowed",
+		SHOULD_CLEAR_CREDENTIALS: "shouldClearCredentials",
 		ANALYTICS_ENDPOINT_ID: "analyticsEndpointId",
 		ANALYTICS_CREDS: "analyticsCreds",
-		PAGE_VIEW_IDENTIFIERS: "pageViewIdentifiers"
+		PAGE_VIEW_IDENTIFIERS: "pageViewIdentifiers",
+		FASTEST_REGION: "fastestRegion",
+		LOCAL_APP_VERSION: "localAppVersion"
 	},
 	ROUTES: {
 		DEFAULT: "/",
 		DEMO: "/demo",
+		NL_DEMO: "/demo?dp=Esri&nl=true",
+		NOT_FOUND: "/404",
+		ERROR_BOUNDARY: "/error",
 		OVERVIEW: "/overview",
-		PRODUCT: "/product",
 		SAMPLES: "/samples",
 		SAMPLE_DETAILS: "/sample/:sampleId",
 		HELP: "/demo/help",
 		TERMS: "/demo/terms",
-		SOFTWARE_ATTRIBUTIONS: "/showcase/software-attributions",
-		BLOG: "/blog",
-		NEW: "/new",
-		UNAUTHORIZED: "/401",
-		FORBIDDEN: "/403",
-		NOT_FOUND: "/404",
-		ERROR_BOUNDARY: "/error"
+		SOFTWARE_ATTRIBUTIONS: "/showcase/software-attributions"
```

Comment:
Nit: the value in appConfig.ts is "/demo/software-attributions"

Reply:
thanks @imraymondlee , fixing this right away